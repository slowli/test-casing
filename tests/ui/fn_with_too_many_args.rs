use test_casing::test_casing;

#[test_casing(1, [(1, 2, 3, 4, 5, 6, 7, 8)])]
fn tested_function(
    _arg0: i32,
    _arg1: i32,
    _arg2: i32,
    _arg3: i32,
    _arg4: i32,
    _arg5: i32,
    _arg6: i32,
    _arg7: i32,
) {
    // Does nothing
}

fn main() {}
