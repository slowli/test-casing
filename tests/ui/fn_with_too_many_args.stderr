error: tested function must have no more than 7 args
  --> tests/ui/fn_with_too_many_args.rs:4:1
   |
4  | / fn tested_function(
5  | |     _arg0: i32,
6  | |     _arg1: i32,
7  | |     _arg2: i32,
...  |
12 | |     _arg7: i32,
13 | | ) {
   | |_^
